LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Jesse Berger (user alias: valadian)
```

Comment:
Sorry, no. If the Original Work was Jesse's -- anything anyone else has done is explicitly done under his licensing, under his terms. The only way to change the license would be to get consent from all authors, but especially the author of the Original Work.